### ISSUE TITLE
Not actual version of package

### ISSUE THREAD

**Issue by padlyuck** [webvimark/components#2]
Information about the latest version of the package https://packagist.org/packages/webvimark/components has not refreshed
Packagist support answer
The package does not have the github hook set up for auto-updates so that's the main reason it did not update.

**Comment by webvimark** [comment]
Yeah, have this problem. I set up hook for packagist, but it's not auto-refreshing. Have no idea why :(

P.S. Update it manually right now

**Comment by padlyuck** [comment]
Try to add https://packagist.org as a domain in packagist service configuration. GitHub default is http://packagist.org (http instead of https)

**Comment by webvimark** [comment]
Tried it, doesn't work. But if you have some ideas I'll try them, because tired of manual updating )
